libs/platform/src/lib/components/list/base-list-item.ts

```diff
@@ -0,0 +1,370 @@
+    @Input()
+    description: string;
+
+    /** Tooltip text to show when focused for more than timeout value*/
+    @Input()
+    title?: string;
+
+    /**radio button value */
+    @Input()
+    value: string;
+
+    /**
+    * Enabling this flag causes forcing title directive to not wrap text,
+    * instead of wrapping there will be text truncation
+    */
+    @Input()
+    titleWrap?: boolean;
+
+    /** attribute to hold primary/title icon*/
+    @Input()
+    titleIcon?: string;
+
+    /** attribute to hold secondary text*/
+    @Input()
+    secondary?: string;
+
+    /** attribute to hold counter value*/
+    @Input()
+    counter?: string;
+
+    /** attribute to hold avatar path*/
+    @Input()
+    avatarSrc?: string;
+
+    /** attribute to hold avatar title for a11y*/
+    @Input()
+    avatarTitle: string;
+
+    /**
+     * Enabling this flag causes forcing secondary item directive to not wrap text,
+     * instead of wrapping there will be text truncation
+     */
+    @Input()
+    secondaryWrap?: boolean;
+
+    /** attribute holds secondary icon value*/
+    @Input()
+    secondaryIcons?: SecondaryActionItem[];
+
+    /** define label for screen reader */
+    @Input()
+    ariaLabelledBy: string;
+
+    /** define level of item for screen reader */
+    @Input()
+    ariaLevel: number;
+
+    /** define position of item for screen reader */
+    @Input()
+    ariaPosinet: number;
+
+    /** Whether Navigation mode is included to list component
+    * for all the items
+    */
+    navigated = false;
+
+    /** Whether Navigation mode is included to list component
+     * only a subset of the list items are navigable
+     * you should indicate those by displaying a navigation arrow
+    */
+    navigationIndicator = false;
+
+    /**By default selection mode is 'none' */
+    selectionMode: SelectionType = 'none';
+
+    /**By default selection mode is 'active' */
+    listType: ListType;
+
+    /** Used for placeing navigation Link */
+    link: string;
+
+    noSeperator: boolean;
+
+    /** Whether By line mode is included to list component, by which
+     *  list item will accomdate the data in 2 column
+     */
+    hasByLine = false;
+
+    /** @hidden */
+    _contentDensity = this._listConfig.contentDensity;
+
+
+    /** Whether listitem is selected */
+    selected: boolean;
+
+    public selectionValue: string;
+
+    /**
+     * @hidden
+     * Used to define if contentDensity value is 'compact' or not.
+     */
+    isCompact = this._contentDensity === 'compact';
+
+
+    /** event emitter for selected item*/
+    @Output()
+    itemSelected: EventEmitter<any> = new EventEmitter<any>();
+
+    /** Event sent when delete, details or any other action buttons are clicked */
+    @Output()
+    buttonClicked: EventEmitter<any> = new EventEmitter();
+
+    @ViewChild('listItem')
+    listItemRef: ElementRef;
+
+    @ViewChild('listItem', { read: ElementRef, static: false })
+    listItem: ElementRef;
```

Comment:
Why do you have 2 same references there? Also let's remove `static: false`, since it's default one.

Reply:
updated